snake_test.go

```diff
@@ -88,7 +90,7 @@ func toSnakeWithIgnore(tb testing.TB) {
 		{"ManyManyWords", "many_many_words"},
 		{"manyManyWords", "many_many_words"},
 		{"AnyKind of_string", "any_kind_of_string"},
-		{"numbers2and55with000", "numbers_2_and_55_with_000"},
+		{"numbers2and55With000", "numbers2and55_with000"},
```

Comment:
Same here - the input is changed from `numbers2and55with000` to `numbers2and55With000`.